tools/d1-migration/add-to-dynamo.js

```diff
@@ -0,0 +1,240 @@
+import {
+  DynamoDBClient,
+  BatchWriteItemCommand
+} from '@aws-sdk/client-dynamodb'
+import { marshall } from '@aws-sdk/util-dynamodb'
+import { CBOR } from '@ucanto/server'
+
+import { exec as childProcessExec } from 'child_process'
+
+/**
+ * 
+ * @param {string} command 
+ * @returns 
+ */
+const exec = async (command) => {
+  return new Promise((resolve, reject) => {
+    childProcessExec(command, (error, stdout, stderr) => {
+      if (error !== null) reject(error)
+      if (stderr !== '') reject(stderr)
+      else resolve(stdout)
+    })
+  })
+}
+
+async function loadFromD1 () {
+  const {
+    STAGE,
+  } = getEnv()
+
+  const dbName = (STAGE === 'prod') ? 'access' : 'access-staging'
+  try {
+    const delegations = JSON.parse(await exec(`wrangler d1 execute ${dbName} --command 'SELECT * from delegations_v3' --json`))[0].results
```

Comment:
Code comment on why we need `[0].results` would be very welcome.

Reply:
done!